src/guiguts/file.py

```diff
@@ -447,6 +475,21 @@ def get_current_page_label(self) -> str:
         else:
             return self.page_details[img]["label"]
 
+    def set_languages(self) -> None:
+        """Allow the user to set languages.
+
+        Multiple languages may be separated by any non-word character
+        which will be converted to "+"
+        """
+        languages = simpledialog.askstring(
+            "Set Language(s)", "Language(s)", parent=maintext()
+        )
+        if languages:
+            lang_list = re.split(r"\W", languages)
```

Comment:
This could do interesting things if someone inputs a locale (`en_US`) here instead of a language.

Reply:
I think it's OK, because `\W` means not `\w` (alphanumeric plus underscore character). I did think about that when I used `\W` and thought that it would be OK for locales.

However, I didn't take that into account when I checked if English was one of the languages in the Title Case check, so I'll do something about that now.